src/applications/facility-locator/actions/index.js

```diff
@@ -119,58 +151,35 @@ export const searchWithBounds = ({ bounds, facilityType, serviceType, page = 1 }
         if (!address) {
           dispatch({
             type: SEARCH_FAILED,
-            // eslint-disable-next-line prettier/prettier
-            error: 'Reverse geocoding failed. See previous errors or network log.',
+            error:
+              'Reverse geocoding failed. See previous errors or network log.',
           });
           return;
         }
 
-        // eslint-disable-next-line prettier/prettier
-        fetchLocations(address, bounds, facilityType, serviceType, page, dispatch);
+        fetchLocations(
+          address,
+          bounds,
+          facilityType,
+          serviceType,
+          page,
+          dispatch,
+        );
       });
     } else {
-      // eslint-disable-next-line prettier/prettier
       fetchLocations(null, bounds, facilityType, serviceType, page, dispatch);
     }
   };
 };
 
-/**
- * Handles the actual API call to get the type of locations closest to `address`
- * and/or within the given `bounds`.
- *
- * @param {string=} address Address of the center-point of the search area
- * @param {number[]} bounds Geo-coords of the bounding box of the search area
- * @param {string} locationType (see config.js for valid types)
- * @param {string} serviceType (see config.js for valid types)
- * @param {number} page What page of results to request
- * @param {Function} dispatch Redux's dispatch method
- */
-// eslint-disable-next-line prettier/prettier
-const fetchLocations = async (address = null, bounds, locationType, serviceType, page, dispatch) => {
-
-  try {
-    // eslint-disable-next-line prettier/prettier
-    const data = await LocatorApi.searchWithBounds(address, bounds, locationType, serviceType, page);
-    if (data.errors) {
-      dispatch({ type: SEARCH_FAILED, error: data.errors });
-    } else {
-      dispatch({ type: FETCH_LOCATIONS, payload: data });
-    }
-  } catch (error) {
-    dispatch({ type: SEARCH_FAILED, error });
-  }
-};
-
```

Comment:
I had to move this function above `searchWithBounds`. I think everything else was auto-fixed.